apps/nextjs-website/src/components/organisms/WebinarsSection/WebinarsSection.tsx

```diff
@@ -1,7 +1,7 @@
 'use client';
 import { Webinar } from '@/lib/types/webinar';
 import { Alert, Box, Snackbar, Typography, useTheme } from '@mui/material';
-import React, { useState } from 'react';
+import React, { useMemo, useState } from 'react';
```

Comment:
`useMemo` is not used.

When you are going to remove it, please remove also `import { DevPortalUser } from '@/lib/types/auth';` which is not used too